packages/config/__tests__/__resources__/ProfInfoApp_team_config_proj/ProfInfoApp.config.json

```diff
@@ -57,7 +57,6 @@
         "LPAR4": {
             "type": "dummy",
             "properties": {
-                "host": "LPAR4.your.domain.net",
```

Comment:
Is the removal of the "host" property here intentional?

Reply:
Host property is added to the file again